### PR TITLE
Add reconcile helm commands

### PR DESCRIPTION
**Pull request by stefanprodan**
Changes:
- add `tk reconcile source helm`
- add `tk reconcile hr`
- update tk docs

### CODE REVIEW

File: cmd/tk/reconcile_helmrelease.go [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+	err = kubeClient.Get(ctx, namespacedName, &helmRelease)
+	if err != nil {
+		return err
+	}
+
+	if helmRelease.Status.LastAppliedRevision != "" {
+		logger.Successf("reconciled revision %s", helmRelease.Status.LastAppliedRevision)
+	} else {
+		return fmt.Errorf("HelmRelease sync failed")
```
Comment (hiddeco) [inline cmd/tk/reconcile_helmrelease.go:117]:
```suggestion
		return fmt.Errorf("HelmRelease reconciliation failed")
```

File: cmd/tk/reconcile_helmrelease.go [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+			return err
+		}
+	} else {
+		logger.Actionf("annotating HelmRelease %s in %s namespace", name, namespace)
+		if helmRelease.Annotations == nil {
+			helmRelease.Annotations = map[string]string{
+				helmv2.ReconcileAtAnnotation: time.Now().String(),
+			}
+		} else {
+			helmRelease.Annotations[helmv2.ReconcileAtAnnotation] = time.Now().String()
```
Comment (hiddeco) [inline cmd/tk/reconcile_helmrelease.go:93]: Noticed the other day that the string this creates is kind of awkward: `2009-11-10 23:00:00 +0000 UTC m=+0.000000001`.

Comment (stefanprodan) [reply cmd/tk/reconcile_helmrelease.go:93]: Should we use `metav1.Now().String()` like we do in KS/HC?

Comment (hiddeco) [reply cmd/tk/reconcile_helmrelease.go:93]: Does that print something else? As `metav1.Now()` is just a wrapper around `time` to able to (un)marshal.

Comment (stefanprodan) [reply cmd/tk/reconcile_helmrelease.go]: I've use RFC3339Nano

**Pull request merged**